Sources/Orbit/Components/SegmentedSwitch.swift

```diff
@@ -0,0 +1,214 @@
+import SwiftUI
+
+/// Showing two choices from which only one can be selected.
+///
+/// Currently SegmentedSwitch allows to have 2 segments.
+/// SegmentedSwitch can be in error state only in unselected state.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/interaction/segmentedswitch/)
+public struct SegmentedSwitch: View {
+
+    @Binding private var selectedIndex: Int?
+    let label: String
+    let message: Message?
+    let firstOption: String
+    let secondOption: String
+
+    let separatorWidth: CGFloat = BorderWidth.emphasis
```

Comment:
separator will need to have the same size as border, so it does not have to be specified.

Only the optional error outline should use the BorderWidth.emphasis size, otherwise it should be 2pt as specified in design.

Reply:
I've removed `separatorWidth` but left `borderWidth = BorderWidth.emphasis`. If there is an error this is the correct width, and when there is no error, than borderColor is equal to backgroundColor so borderWidth doesn't matter.

Reply:
true, but the non-error separator still needs 2pt width.

Reply:
in Figma separator is always 2pts in all states. I will use the contstant there 👌